python/src/KrigingAlgorithm_doc.i.in

```diff
@@ -5,8 +5,7 @@ Refer to :ref:`kriging`.
 
 Available constructors:
     KrigingAlgorithm(*inputSample, outputSample, covarianceModel, basis*)
-
-    KrigingAlgorithm(*inputSample, outputSample, covarianceModel, basisCollection*)
+    KrigingAlgorithm(*inputSample, outputSample, covarianceModel*)
```

Comment:
no need for a ctor list when they are nested, I think you can remove the whole list and just mention that the basis argument is optional

Reply:
ok